curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/653639d63a45a077333312c8.md

```diff
@@ -0,0 +1,645 @@
+---
+id: 653639d63a45a077333312c8
+title: Step 17
+challengeType: 0
+dashedName: step-17
+---
+
+# --description--
+
+Before playing the song, you need to make sure it starts from the beginning. This can be achieved by the use of the `currentTime` property of the `audio` object. 
+
+Add an `if` statement that will check whether the `userData.currentSong` is `null` or if the `id` of the current song `userData.currentSong.id` is different from the `id` of the selected song `song.id`. If either of these conditions is true, set the `currentTime` property of the `audio` object to `0`.
+
+Inside the `else` block, set the `audio.currentTime` to `userData.songCurrentTime`.
```

Comment:
I think this should be broken up into two steps 

One step to add the if statement

One step to add the else statement

We are still pretty early on into the curriculum and I think the beginners will benefit more if this was broken up into two steps